### ISSUE TITLE
"Team" should be included in "accessibility" email

### ISSUE THREAD

**Issue by joshkimux** [AIGAdc/dcdesignweek.org-11ty#192]
## Description

"team" isn't wrapped within the email link, adding it in will help a bit!

<img width="621" alt="Screen Shot 2022-09-29 at 8 00 25 PM" src="https://user-images.githubusercontent.com/14154792/193162396-d1aa1dce-8df8-4673-a761-cf3fe1279a19.png">

## Success criteria

- [x] Email link reads as "Email our accessibility team" (and not just "Email our accessibility")
